frontend/src/Store/Actions/Settings/importListOptions.js

```diff
@@ -0,0 +1,64 @@
+import { createAction } from 'redux-actions';
+import createFetchHandler from 'Store/Actions/Creators/createFetchHandler';
+import createSaveHandler from 'Store/Actions/Creators/createSaveHandler';
+import createSetSettingValueReducer from 'Store/Actions/Creators/Reducers/createSetSettingValueReducer';
+import { createThunk } from 'Store/thunks';
+
+//
+// Variables
+
+const section = 'settings.importListOptions';
+
+//
+// Actions Types
+
+export const FETCH_IMPORT_LIST_OPTIONS = 'settings/importListOptions/fetchImportListOptions';
+export const SAVE_IMPORT_LIST_OPTIONS = 'settings/importListOptions/saveImportListOptions';
+export const SET_IMPORT_LIST_OPTIONS_VALUE = 'settings/importListOptions/setImportListOptionsValue';
+
+//
+// Action Creators
+
+export const fetchImportListOptions = createThunk(FETCH_IMPORT_LIST_OPTIONS);
+export const saveImportListOptions = createThunk(SAVE_IMPORT_LIST_OPTIONS);
+export const setImportListOptionsValue = createAction(SET_IMPORT_LIST_OPTIONS_VALUE, (payload) => {
+  return {
+    section,
+    ...payload
+  };
+});
+
+//
+// Details
+
+export default {
```

Comment:
All of the other `Store/Actions/Settings/*.js` things appear to use a big `default` block, as it seems they're all just imported together in `settingsActions.js` rather than separately. I can still make them separate if you wish (just checking before I make it inconsistent)